fuse-android/src/test/java/com/github/kittinunf/fuse/android/FuseByteCacheTest.kt

```diff
@@ -52,7 +52,7 @@ class FuseByteCacheTest : BaseTestCase() {
         assertThat(value, notNullValue())
         assertThat(value!!.toString(Charset.defaultCharset()), equalTo("world"))
         assertThat(error, nullValue())
-        assertThat(source, equalTo(Cache.Source.ORIGIN))
+        assertThat(source, equalTo(Source.ORIGIN))
```

Comment:
Since `Source` was moved to be a top level construct, you will see this change a lot in this PR.

Reply:
👍